app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-page-views.js

```diff
@@ -109,6 +110,17 @@ window.GOVUK.analyticsGa4.analyticsModules = window.GOVUK.analyticsGa4.analytics
       return searchTerm
     },
 
+    getToolName: function () {
+      // This is set by the Ga4SearchTracker module if the user has accepted an autocomplete
+      // suggestion on the search field.
+      if (window.sessionStorage && window.sessionStorage.getItem('searchAutocompleteAccepted')) {
```

Comment:
Is this object detection in case a browser doesn't support window.sessionStorage? it seems such a supported API that this seems a bit superfluous given our browser support (apologies if I've misunderstood).